src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -208,6 +210,20 @@ public GerritTrigger(List<GerritProject> gerritProjects) {
             this.notificationLevel = "";
         }
 
```

Comment:
`silentMode` needs to be deprecated and migrated to `silentLevel`. E.g./I.e. Deprecate `isSilendMode` and replace the implementation with `return silentLevel != null && silentLevel == SilentLevel.ALL` add a `readResolve` and deprecate and set `silentMode` to be `transient` etc.
https://wiki.jenkins-ci.org/display/JENKINS/Hint+on+retaining+backward+compatibility